test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/representee/RepresenteeControllerSpec.scala

```diff
@@ -1191,13 +1391,57 @@ class RepresenteeControllerSpec
               mockStoreSession(session.copy(journeyStatus = Some(newJourney)))(Right(()))
             }
 
-            checkIsRedirect(performAction(formData(NoReferenceId)), routes.RepresenteeController.checkYourAnswers())
+            checkIsRedirect(performAction(formData(NoReferenceId)), routes.RepresenteeController.confirmPersonSubmit())
           }
 
         }
 
       }
 
+      "redirect the page to enter-id" when {
```

Comment:
these tests should already exist in the section of the spec testing the cya ednpoint

Reply:
Done.